<launch>
	<node pkg="auv_codes2" name="only_depth" type="only_depth.py" output="screen" />
	<node pkg="auv_codes2" name="only_straight" type="only_straight.py" output="screen" />
	<node pkg="auv_codes2" name="gui" type="gui.py" output="screen" />


	<node name="serial_node"        pkg="rosserial_python"      type="serial_node.py">
    <param name="port"              type="string"               value="/dev/ttyACM0"/> #port change toh idhar bhi change
    <param name="baud"              type="int"                  value="57600"/> #baud rate bhi dekh lena
    </node>


    <node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" output="screen" />
</launch>
